pkg/scaffold/controller/controllertest.go

```diff
@@ -113,7 +114,12 @@ func TestReconcile(t *testing.T) {
 	defer close(StartTestManager(mgr, g))
 
 	// Create the {{ .Resource.Kind }} object and expect the Reconcile{{ if .Resource.CreateExampleReconcileBody }} and Deployment to be created{{ end }}
-	g.Expect(c.Create(context.TODO(), instance)).To(gomega.Succeed())
+	err = c.Create(context.TODO(), instance)
+	// If getting an Invalid error, you should
+	// 1) modify the "instance" object to make it valid e.g. adding required fields.
+	// 2) remove the following if statement.
+	if apierrors.IsInvalid(err) { return }
```

Comment:
Lets use `t.Logf` to log the above comments as warning messages so that user can discover that he/she needs to improve these tests.

Reply:
Good idea.